.github/workflows/check_pr_description.yml

```
@@ -0,0 +1,66 @@
+name: Checking PR description
```

```suggestion
name: Check PR description
```

Prefer phrasing workflow descriptions as commands just like file names and function names

---

Fine, will take care of it from now onwards. Updated it.